Cargo.toml

```diff
@@ -33,6 +33,7 @@ version = "0.5.3"
 # we publish, otherwise crates.io complains as it doesn't know the version.
 [workspace.dependencies]
 cxx-qt = { path = "crates/cxx-qt" }
+cxx-qt-macro = { path = "crates/cxx-qt-macro" }
```

Comment:
can you split this split into a separate pull request. As the trait stuff we might want to see things develop a little bit further to check things work before merging. And it eases review if things are small

Reply:
Done